### DISCUSSION TITLE
Suggested feature: Display Existing Additional Attributes

### DISCUSSION THREAD

**Discussion by Mathiasw** [keepassxreboot/keepassxc#9999] [Ideas]
I would like it if, when you select "Additional Attributes", you could pick and choose from the existing created attributes across the database, as well as adding new ones. I think the original Keepass does this.

That way, it stops me from forgetting whether to create "E-Mail" or "EMail",  "UserName" or "U-Name" or "UName" so I have consistent additional attributes across the database.

... as if you didn't have ENOUGH people suggesting new features!

Also sent you 50 euros this morning as a conation, it's a top product, but like an idiot - it's really early in the morning here - I forgot to say who it was from!

Thanks for an excellent product, well worth the money.

... Mathias

**Comment by droidmonkey** [comment]
Howdy! This is tracked in #3636. Thank you for the suggestion and donation 😀
